lib/Option/SanitizerOptions.cpp

```diff
@@ -95,9 +100,17 @@ llvm::SanitizerCoverageOptions swift::parseSanitizerCoverageArgValue(
   return opts;
 }
 
+static bool isTsanSupported(
+    const llvm::Triple &Triple,
+    std::function<bool(std::string)> sanitizerRuntimeLibExists) {
+
+  return Triple.isArch64Bit() && sanitizerRuntimeLibExists("tsan");
+}
+
 SanitizerKind swift::parseSanitizerArgValues(const llvm::opt::Arg *A,
-                                      const llvm::Triple &Triple,
-                                      DiagnosticEngine &Diags) {
+                  const llvm::Triple &Triple,
+                  DiagnosticEngine &Diags,
+                  std::function<bool(std::string)> sanitizerRuntimeLibExists) {
```

Comment:
Nitpick: This isn't your fault, but if you can't align parameters then the preference is to double-indent them and leave it, not right-align the longest parameter and then match the others.